src/main/kotlin/com/terraformation/backend/i18n/Messages.kt

```diff
@@ -102,7 +105,28 @@ class Messages {
           title = "$deviceName cannot be detected.",
           body = "$deviceName cannot be detected. Please check on it.")
 
+  fun historyAccessionCreated() = "created accession"
+
+  fun historyAccessionStateChanged(newState: AccessionState) =
+      "updated the status to ${newState.displayName}"
```

Comment:
would we need a `from` state since user will be able to update state to any value (doesn't have to follow logically)

Reply:
The history list will always have all the previous transitions, so they should be able to infer the "from" state by looking at the preceding state change.

Reply:
yes, makes sense